scripts/deploy.js

```diff
@@ -20,16 +20,9 @@ var Secrets = function() {
 };
 
 Secrets.prototype.load = function () {
-  var config = fs.readFileSync('secrets.json');
-
-  if (config) {
-    config = JSON.parse(config);
-    this.bucket = config.bucket;
-    AWS.config.credentials = new AWS.Credentials(config.access_key_id, config.secret_access_key);
-    AWS.config.region = 'us-east-1';
-  } else {
-    console.error('Missing AWS secrets. You must decrypt the ejson first.');
-  }
+  this.bucket = process.env.AWS_BUCKET;
+  AWS.config.credentials = new AWS.Credentials(process.env.AWS_ACCESS_KEY_ID, process.env.AWS_SECRET_ACCESS_KEY);
+  AWS.config.region = 'us-east-1';
```

Comment:
Yeah. Any changes needed to be done to these files can me committed directly to master.